### PR TITLE
chore: Refine readme and replace docs with docs.pgvecto.rs

### LINKED ISSUE #233: chore: Remove docs from the repo, keeping docs in pgvecto.rs-docs
I am working on our doc website. After it works, we should remove docs from the repo, to reduce the maintenance cost. Let's maintain our doc there.

### PR DESCRIPTION
**Pull request by gaocegege**
Close #233 
Ref #214

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,16 +35,11 @@ More details at [./docs/comparison-pgvector.md](./docs/comparison-pgvector.md)
 
 ## Documentation
 
-- [Installation](./docs/installation.md)
```
Comment (usamoi) [inline README.md:38]: Can we still keep these links in README?

**Review by usamoi — COMMENTED** [review]
Please fix broken links in https://github.com/tensorchord/pgvecto.rs/blob/main/crates/service/src/prelude/error.rs

**Comment by gaocegege** [comment]
Comments addressed. Preview is here: https://github.com/gaocegege/pgvecto.rs/tree/docs

PTAL

File: README.md [hunk 2]
```diff
@@ -31,20 +31,24 @@ pgvecto.rs is a Postgres extension that provides vector similarity search functi
 | Async Index build                           | Ready for queries anytime and do not block insertions. | ❌                       |
 | Quantization                                | Scalar/Product Quantization                            | ❌                       |
 
-More details at [./docs/comparison-pgvector.md](./docs/comparison-pgvector.md)
+More details at [pgvecto.rs vs. pgvector](https://docs.pgvecto.rs/faqs/comparison-pgvector.html)
 
-## Documentation
+## [Documentation]((https://docs.pgvecto.rs/getting-started/overview.html))
```
Comment (usamoi) [inline README.md:36]: The link does not work.

File: crates/service/src/prelude/error.rs [hunk 3]
```diff
@@ -65,17 +65,17 @@ ADVICE: Check if dimensions and scalar type of the vector is matched with the in
...
     #[error("\
 The extension is upgraded so all index files are outdated.
-ADVICE: Delete all index files. Please read `https://github.com/tensorchord/pgvecto.rs/blob/main/docs/upgrade.md`.\
+ADVICE: Delete all index files. Please read `https://docs.pgvecto.rs/faqs/upgrading.html`.\
 ")]
     Upgrade,
     #[error("\
 The extension is upgraded so this index is outdated.
-ADVICE: Rebuild the index. Please read `https://github.com/tensorchord/pgvecto.rs/blob/main/docs/upgrade.md`.\
+ADVICE: Rebuild the index. Please read `https://docs.pgvecto.rs/faqs/upgrading.html`.\
```
Comment (usamoi) [inline crates/service/src/prelude/error.rs:78]: Can we move it from `faqs` to `admin`?

Comment (gaocegege) [reply crates/service/src/prelude/error.rs:78]: SGTM.

Comment (gaocegege) [reply crates/service/src/prelude/error.rs:78]: SGTM.

**Comment by gaocegege** [comment]
The upgrading doc is moved to admin, and README is updated.

**Review by usamoi — APPROVED** [review]

**Pull request merged**